README.md

```diff
@@ -0,0 +1,56 @@
+# Azure SDK for Python
+
+[![Python](https://img.shields.io/pypi/pyversions/azure-core.svg?maxAge=2592000)](https://pypi.python.org/pypi/azure/) [![BuildStatus](https://dev.azure.com/azure-sdk/public/_apis/build/status/46?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=46&branchName=master) [![Build](https://travis-ci.org/Azure/azure-sdk-for-python.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-python) [![Dependencies](https://img.shields.io/badge/dependencies-analyzed-blue.svg)](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-python/dependencies/dependencies.html)
+
+This repository contains official Python libraries for Azure services. For documentation go to [Azure SDK for Python documentation](http://aka.ms/python-docs).
+
+You can find a complete list of all the packages for these libraries [here](./packages.md).
+
+## Getting started
+
+For your convenience, each service has a separate set of libraries that you can choose to use instead of one, large Azure package. To get started with a specific library, see the `README.md` (or `README.rst`) file located in the library's project folder.
+
+You can find service libraries in the `/sdk` directory.
+
+### Prerequisites
+The client libraries are supported on Python 2.7 and 3.5.3 or later.
```

Comment:
We officially kill PyPy?

Reply:
Would you expect PyPy to be called out explicitly here? I didn't call out CPython... Btw, the existing README.rst does not call out PyPy explicitly...

Reply:
Ok, I get your point. Then we lie a little, if someone asks for `azure-servicebus` support arguing that we see we support 3.5.3 and later, what is the answer then? Valid bug? Valid feature request? Not gonna happen?

If you suggest this sentence says "Whatever Python implementation that is compatible with 2.7 and 3.5.3 and higher is supported", then that's not actually true :(. I could just mention Jython or IronPython, etc.

Reply:
Note: I'm not suggesting blocking your PR for this point, I just feel it deserves discussion and there is possible issue opened by people that we should at least be prepared to answer.

Reply:
I'm comfortable with this statement.